tests/unit/moduleapi/cluster.tcl

```diff
@@ -16,12 +16,26 @@ start_cluster 3 0 [list config_lines $modules] {
         assert_equal OK [$node1 test.pingall]
         assert_equal 2 [CI 0 cluster_stats_messages_module_sent]
         wait_for_condition 50 100 {
-            [CI 1 cluster_stats_messages_module_received] eq 1 &&
-            [CI 2 cluster_stats_messages_module_received] eq 1
+            [CI 1 cluster_stats_messages_module_received] eq 2 &&
+            [CI 2 cluster_stats_messages_module_received] eq 2
```

Comment:
This is because when you receive `DING` you send out another message in the receiver. Check my comment on the module code.